src/vis_cpu/conversions.py

```diff
@@ -22,3 +25,149 @@ def lm_to_az_za(el, m):
     az = -np.arctan2(m, el)
     za = np.pi / 2.0 - np.arcsin(n)
     return az, za
+
+
+def point_source_crd_eq(ra, dec):
+    """
+    Calculate approximate coordinate transform of source locations from 
+    equatorial to Cartesian.
+    
+    Parameters
+    ----------
+    ra, dec : array_like
+        1D arrays of source positions in equatorial coordinates (radians).
+    
+    Returns
+    -------
+    array_like
+        Equatorial coordinates of Healpix pixels, in Cartesian
```

Comment:
Let's remove references to healpix, I think. The positions are arbitrary.

Reply:
Done.